### DISCUSSION TITLE
Question for coders

### DISCUSSION THREAD

**Discussion by Appfanatic456** [community/community#32288] [New to GitHub]
I'm new to coding so I don't have much experience. I want to create and build a new website on my own. Does anyone have any opinion on what code I should use? 

Like is one code easier or better than the other? The 2 coding languages I'm considering are C++ and Java.

**Comment by C3n7ral051nt4g3ncy** [comment]
@Appfanatic456, 

Java is easier to learn first. 
I would go with `Java`.

Best of luck!

**Reply by Adwazabeel** [reply]
Pleas full deleat

**Comment by Gabri432** [comment]
In my opinion, if you are new to coding and you would like to build a website, you should learn HTML, CSS and JavaScript.
Then you could use Github Pages to host your own static website for free.

But between the two languages you are referring I think Java would be better.

**Comment by garydsong** [comment]
A lot of coding bootcamps teach JavaScript first then segue into other languages to tie everything together such as HTML, CSS, SQL, and Python.  If your end goal is to build websites and applications you may want to become accustomed to JavaScript since React is so popular.  React uses JavaScript and JSX, so going the JS, HTML, CSS route is probably most beneficial for what you want.

**Comment by surajkrsingh** [comment]
WordPress is an easy platform to create and live a site within a few minutes if you need your site too quickly. 
This is very easy to use and you don't need more technical knowledge to handle it, Everything can be added using the plugin and free theme available in WordPress Directory.
And if you learn or know the code, you can improve it later or can make it better.

**Comment by macknive** [comment]
If your target is to create your own website the tools you will need is html css and js.

**Comment by avwerosuotrust** [comment]
If you are passionate about creating your own website, you will have to start with the basic languages of building a website
and they are
1. HTML
2. CSS
3. JAVASCRIPT  
You can learn these languages for free using Freecodecamp.
